tests/controllers/user.test.js

```diff
@@ -3,221 +3,349 @@ const supertest = require('supertest')
 const server = require('../../src/server.js')
 const User = require('../../src/models/User.js')
 const { dropCollectionAndEnd } = require('../helpers/index.js')
-const { validUser1, validUser2, invalidUser1 } = require('../fixtures/users.json')
+const { validUser1, validUser2, invalidUser1, validAdminUser } = require('../fixtures/users.json')
+
+const { makeLoggedInToken } = require('../../src/controllers/session.js')
 
 // Tests for: GET /users
-tape('GET /users when nothing in database', t => {
+tape('GET /users when not logged in', t => {
   supertest(server)
     .get('/users')
-    .expect(200)
+    .expect(401)
     .expect('Content-Type', /json/)
-    .end((err, res) => {
-      if (err) t.fail(err)
-      t.equal(res.body.length, 0, 'should initially return empty array')
-      dropCollectionAndEnd(User, t)
+    .then(res => {
+      t.equal(res.body.message, 'You are not authorised to access this resource', 'unauthorised error message sent')
```

Comment:
import the message from constants file?

Reply:
👍  yeah I was being lazy, can do that now